packages/wonder-blocks-timing/src/components/with-action-scheduler.tsx

```diff
@@ -18,11 +18,20 @@ type WithoutActionScheduler<T> = Omit<T, "schedule">;
 export default function withActionScheduler<
     Props extends WithActionSchedulerProps,
 >(WrappedComponent: React.ComponentType<Props>) {
-    return (props: WithoutActionScheduler<Props>) => (
+    const displayName = `withActionScheduler(${
+        WrappedComponent.displayName || WrappedComponent.name
+    })`;
+
+    const C = (props: WithoutActionScheduler<Props>) => (
         <ActionSchedulerProvider>
             {(schedule: IScheduleActions) => (
                 <WrappedComponent {...(props as Props)} schedule={schedule} />
             )}
         </ActionSchedulerProvider>
     );
+
+    C.displayName = displayName;
+    C.WrappedComponent = WrappedComponent;
```

Comment:
**question:** Is exposing this bit necessary?

Reply:
Possibly not. It seems to be part of the pattern for HoCs in fairly core React libs (Redux, for example, [does this also](https://github.com/reduxjs/react-redux/blob/4a7e129ee537d35c53da258ccf7924a0376391ee/src/components/connect.tsx#L785)).

However, given I'm unsure what it does, I'm happy to remove this and just set the `.displayName`

Reply:
We use the `.displayName` part in our code a lot. Not seen the `WrappedComponent` piece; I wonder if that's there for their debug tools or something.

Reply:
Removed for now. If we find out it helps, we can always do another PR.